process/sync/metablock.go

```diff
@@ -60,6 +60,11 @@ func NewMetaBootstrap(arguments ArgMetaBootstrapper) (*MetaBootstrap, error) {
 		uint64Converter:     arguments.Uint64Converter,
 		poolsHolder:         arguments.PoolsHolder,
 		indexer:             arguments.Indexer,
+		isInImportMode:      arguments.IsInImportMode,
+	}
+
+	if arguments.IsInImportMode {
```

Comment:
Maybe to be sure that isImportMode was correctly or was not missed to be set with the value of argument.IsImportMode you can check directly with isInImportMode in this condition

Reply:
done